packages/connect/src/api/signMessage.ts

```diff
@@ -55,6 +53,17 @@ export default class SignMessage extends AbstractMethod<'signMessage', PROTO.Sig
         };
     }
 
+    get info() {
+        // FIXME!!!!
+        let coinInfo: BitcoinNetworkInfo | undefined;
+        if (this.payload.coin) {
+            coinInfo = getBitcoinNetwork(this.payload.coin);
+        } else {
+            coinInfo = getBitcoinNetwork(this.payload.path);
+        }
```

Comment:
Same as this? Also, are we sure that `validatePath(payload.path)` is now unnecessary?
```suggestion
        const coinInfo = getBitcoinNetwork(this.payload.coin ?? this.payload.path);
```

Reply:
imho it doesn't matter now. the method would die on validation in init. also changed - not using `this.payload.path` but this `this.params.address_n` instead which is result from `validatePath(payload.path);`